portfolio/src/main/java/com/google/sps/servlets/BlobstoreUploadURLServlet.java

```diff
@@ -0,0 +1,26 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.blobstore.BlobstoreService;
+import com.google.appengine.api.blobstore.BlobstoreServiceFactory;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * When the fetch() function requests the /blobstore-upload-url URL, the content of the response is
+ * the URL that allows a user to upload a file to Blobstore. 
+ */
+@WebServlet("/blobstore-upload-url")
+public class BlobstoreUploadURLServlet extends HttpServlet {
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    BlobstoreService blobstoreService = BlobstoreServiceFactory.getBlobstoreService();
+    String uploadUrl = blobstoreService.createUploadUrl("/add-comment");
+
+    response.setContentType("text/html");
```

Comment:
test/html can be a constant. In fact we are using numerous content types in various files so we can declare these in a separate constants file and use the same everywhere

Reply:
Yes you're right, that was a minor oversight on my part. I'll make that change!